tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -307,5 +422,26 @@ class CompletionMatchSuite extends BaseCompletionSuite {
        |}
        |}""".stripMargin,
     filter = _.contains("exhaustive"),
+    compat = Map(
+      "3" -> s"""
+                |package example
+                |
+                |sealed trait TestA
+                |case object Foo extends TestA
+                |case object Bar extends TestA
+                |sealed trait TestB
+                |case object Baz extends TestB
+                |case object Goo extends TestB
+                |
+                |object Main {
+                |  def testExhaustive[T <: TestA with TestB](test: T): Boolean =
+                |    test match
+                |\tcase Foo => $$0
+                |\tcase Bar =>
+                |\tcase Baz =>
+                |\tcase Goo =>
+                |
+                |}""".stripMargin
+    ),
```

Comment:
Can you add a test for Scala3 enum exhaustive match?

Reply:
Sure, I forgot about that. Also, currently we are importing all enum cases separately, maybe for enums we should do sth like `case Color.Red` instead of `import Color.Red`?

Reply:
You meant to ` case Color` instead of import Color.Red` ?
Yes, it would be better with that :) However, we can do that in follow-up PR :+1: